### ISSUE TITLE
Configure apm install aliases (apm i)

### ISSUE THREAD

**Issue by icyflame** [atom/apm#435]
- Alias `apm i` to `apm install`

**Comment by jwhitmarsh** [comment]
:+1: I know I could do this with a custom alias, but it'd be nice if it came with it

**Comment by icyflame** [comment]
Indeed, James. We could do it with an alias, but since this alias is a part
of the npm install, I just expected it to be there. That kind of uniformity
would be awesome to have!

**Comment by bronson** [comment]
Anyone want to give #484 a look?

**Comment by icyflame** [comment]
@bronson Thanks!

Closing this in favor of #484.
